src/app/helptext/storage/volumes/datasets/dataset-acl.ts

```diff
@@ -157,5 +157,8 @@ dataset_acl_toplevel_dialog_message: T('Editing top-level datasets can\
 dataset_acl_add_item_btn: T('Add ACL Item'),
 
 acl_defaults_placeholder: T('Default ACL Options'),
-acl_defaults_tooltip: T('Set default options on this ACL.')
+acl_defaults_tooltip: T('Set default options on this ACL.'),
+
+user_not_found: T('User name could not be resolved.'),
```

Comment:
After talking with Andrew:
```suggestion
user_not_found: T('Could not find a user name for this user ID.'),
```